src/plugins/filedelete/private.h

```diff
@@ -102,8 +102,7 @@
  *                                                                         *
  ***************************************************************************/
 
-#ifndef FILEDELETE_PRIVATE_H
-#define FILEDELETE_PRIVATE_H
```

Comment:
I would prefer the #ifndef/#define style here, just for consistency.

Reply:
Done.